cli/installer/kubernetes.go

```diff
@@ -21,7 +25,258 @@ var kubernetes = installer{
 }
 
 func kubernetesInstaller(config configuration, ui UI) {
-	fmt.Println("********", config)
+	trackInstall("kubernetes", config, nil)
+
+	installCertManager(config, ui)
+	installJaegerOperator(config, ui)
+
+	execCmdIgnoreErrors(kubectlCmd(config, "create namespace "+config.String("k8s.namespace")))
+
+	installJaeger(config, ui)
+	installCollector(config, ui)
+
+	installTracetest(config, ui)
+
+}
+
+func installTracetest(conf configuration, ui UI) {
+	execCmd(
+		helmCmd(conf, "repo add --force-update kubeshop https://kubeshop.github.io/helm-charts"),
+		ui,
+	)
+	execCmd(
+		helmCmd(conf, "repo update"),
+		ui,
+	)
+
+	cmd := []string{
+		"upgrade --install tracetest kubeshop/tracetest",
+		"--namespace " + conf.String("k8s.namespace") + " --create-namespace",
+	}
+
+	if conf.Bool("k8s.expose") {
+		cmd = append(cmd,
+			"--set 'ingress.hosts[0].host="+conf.String("k8s.ingress-host")+
+				",ingress.hosts[0].paths[0].path=/,ingress.hosts[0].paths[0].pathType=Prefix'",
+		)
+	}
+
+	execCmd(helmCmd(conf, cmd...), ui)
+
+	psql := "host=tracetest-postgresql user=tracetest password=not-secure-database-password  port=5432 sslmode=disable"
+	c := getTracetestConfigFileContents(psql, ui, conf)
+	ttc := createTmpFile("tracetest-config", string(c), ui)
+	defer os.Remove(ttc.Name())
+	execCmd(
```

Comment:
I saw you have a `execCmdIgnoreErrors` function. And I see you also use `execCmd` without assigning the returned error. Do you really need that `execCmdIgnoreErrors` function?

Reply:
`execCmd` internally calls `_execCmd` and exists on error. `execCmdIgnoreErrors` calls `_execCmd` and ignores errors. I don't want to directly call `_execCmd`